src/liballoc/vec.rs

```diff
@@ -2701,6 +2701,9 @@ impl<T> ExactSizeIterator for Drain<'_, T> {
     }
 }
 
+#[unstable(feature = "trusted_len", issue = "37572")]
```

Comment:
I think we’ll need to drop a `#[stable]` attribute on here instead, since we can’t land unstable trait impls.

Reply:
`TrustedLen` is still unstable, thus the `#[unstable]` label.

Reply:
See for example [`TrustedLen for slice::Iter`](https://doc.rust-lang.org/src/core/slice/mod.rs.html#3327).

Reply:
Aha! Ok this looks good to me.